Library/ViewModels/BackerDashboardProjectsViewModelTests.swift

```diff
@@ -55,8 +55,8 @@ internal final class BackerDashboardProjectsViewModelTests: TestCase {
       self.emptyStateProjectsType.assertValues([.backed])
       self.isRefreshing.assertValues([true, false])
 
-      self.vm.inputs.viewWillAppear(false)
-      self.isRefreshing.assertValues([true, false, true])
+      self.vm.inputs.viewWillAppear(true)
+      self.isRefreshing.assertValues([true, false, true], "Projects refresh after swipe animation.")
```

Comment:
@mbrandonw so you're right, it jumps back anywhere from 1 to 6 cells back after reloading OR paginating. I didn't see it because I had `DiscoveryParams.lens.perPage` set to `20` for our test account with fewer backed projects. The higher I set this value, the better it seems. Thoughts on how to keep the positioning other than loading more projects up front?